src/librbd/ImageCtx.h

```diff
@@ -158,7 +158,7 @@ namespace librbd {
     uint32_t blacklist_expire_seconds;
     uint32_t request_timed_out_seconds;
     bool enable_alloc_hint;
-    static bool _filter_metadata_confs(const string &prefix, std::map<string, bool> &configs,
+    bool _filter_metadata_confs(const string &prefix, std::map<string, bool> &configs,
```

Comment:
I just think static is OK? static should be prefer because it could add encapsulation to this method